src/content/en/updates/2019/12/devtools.md

```diff
@@ -0,0 +1,232 @@
+</figure>
+
+## Network panel updates {: #network }
+
+### Request Initiator Chains in the Initiator tab {: #initiators }
+
+You can now view the initiators and dependencies of a network request as a nested list. This
+can help you understand why a resource was requested, or what network activity a certain resource
+(such as a script) caused.
+
+<figure>
+  <img src="../../images/2019/12/initiators.png"
+       alt="A screenshot of a Request Initiator Chain in the Initiator tab"/>
+</figure>
+
+
+After [logging network activity in the Network panel](/web/tools/chrome-devtools/network), click
+a resource and then go to the **Initiator** tab to view its **Request Initiator Chain**:
+
+* The *inspected resource* is bold. In the screenshot above, `https://web.dev/default-627898b5.js`
+  is the inspected resource.
+* The resources above the inspected resource are the *initiators*. In the screenshot above,
+  `https://web.dev/bootstrap.js` is the initiator of `https://web.dev/default-627898b5.js`.
+  In other words, `https://web.dev/bootstrap.js` caused the network request for 
+  `https://web.dev/default-627898b5.js`.
+* The resources below the inspected resource are the *dependencies*. In the screenshot above,
+  `https://web.dev/chunk-f34f99f7.js` is a dependency of `https://web.dev/default-627898b5.js`.
+  In other words, `https://web.dev/default-627898b5.js` caused the network request for
+  `https://web.dev/chunk-f34f99f7.js`.
+
+<aside class="note">
+  Initiator and dependency information can also be accessed by holding <kbd>Shift</kbd> and then
+  hovering over network resources. See 
+  <a href="/web/tools/chrome-devtools/network/reference#initiators-dependencies">View
+  initiators and dependencies</a>.
+</aside>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/df0ca90392989452ad997537308e52a9257f3377 #}
+
+Chromium issue [#842488](https://crbug.com/842488)
+
+### Highlight the selected network request in the Overview {: #overview }
+
+After you click a network resource in order to inspect it, the Network panel now puts a blue border 
+around that resource in the **Overview**. This can help you detect if the network request is 
+happening earlier or later than expected.
+
+<figure>
+  <img src="../../images/2019/12/overview.png"
+       alt="A screenshot of the Overview pane highlighting the inspected resource."/>
+</figure>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/3c77a5937675b59056598afd236b22eaeedc679f #}
+
+Chromium issue [#988253](https://crbug.com/988253)
+
+### URL and path columns in the Network panel {: #columns }
+
+Use the new **Path** and **URL** columns in the **Network** panel to see the absolute path
+or full URL of each network resource.
+
+<figure>
+  <img src="../../images/2019/12/columns.png"
+       alt="A screenshot of the new Path and URL columns in the Network panel."/>
+</figure>
+
+Right-click the **Waterfall** table header and select **Path** or **URL** to show the new
```

Comment:
This should work with any column header, not just the waterfall.

Reply:
@ghodir yes that was the old functionality that I was familiar with. Looks like a regression has been introduced, because it currently only works on the **Waterfall**.